src/screens/WasteCollectionScreen.js

```diff
@@ -0,0 +1,299 @@
+    fetchPolicy: addressesFetchPolicy,
+    skip: !addressesRefreshTime
+  });
+
+  const { data: typesData, loading: typesLoading } = useQuery(
+    getQuery(QUERY_TYPES.PUBLIC_JSON_FILE),
+    {
+      variables: { name: 'wasteTypes' },
+      fetchPolicy: typesFetchPolicy,
+      skip: !typesRefreshTime
+    }
+  );
+
+  // only query if we have a street selected
+  const { data: streetData } = useQuery(getQuery(QUERY_TYPES.WASTE_STREET), {
+    variables: { ids: selectedStreetId },
+    fetchPolicy: streetFetchPolicy,
+    skip: !streetRefreshTime || !selectedStreetId
+  });
+
+  const addressesData = data?.wasteAddresses;
+
+  let parsedTypesData;
+  try {
+    if (typesData?.publicJsonFile?.content) {
+      parsedTypesData = JSON.parse(typesData.publicJsonFile.content);
+    }
+  } catch (error) {
+    console.warn(error, data);
+  }
+
+  let usedTypes = {};
+  streetData?.[QUERY_TYPES.WASTE_ADDRESSES]?.[0]?.wasteLocationTypes?.forEach(
+    (wasteLocationType) => {
+      if (parsedTypesData?.[wasteLocationType.wasteType])
+        usedTypes[wasteLocationType.wasteType] = parsedTypesData?.[wasteLocationType.wasteType];
+    }
+  );
+
+  const renderSuggestion = useCallback(
+    ({ item }) => (
+      <TouchableOpacity
+        onPress={() => {
+          setInputValue(getStreetString(item));
+          Keyboard.dismiss();
+        }}
+      >
+        <RegularText>{getStreetString(item)}</RegularText>
+      </TouchableOpacity>
+    ),
+    [setInputValue]
+  );
+
+  // TODO: disabled until backend is ready
+  // const goToReminder = useCallback(
+  //   () => navigation.navigate('WasteReminder', { wasteTypes: usedTypes, streetData }),
+  //   [navigation, usedTypes, streetData]
+  // );
+
+  useEffect(() => {
+    if (!addressesData) {
+      return;
+    }
+
+    const item = addressesData.find(
+      (street) => getStreetString(street).toLowerCase() === inputValue.toLowerCase()
+    );
+
+    setSelectedStreetId(item?.id);
+  }, [inputValue, setSelectedStreetId, addressesData]);
+
+  if (loading || typesLoading) {
+    return (
+      <LoadingContainer>
+        <ActivityIndicator color={colors.accent} />
+      </LoadingContainer>
+    );
+  }
+
+  const filteredStreets = filterStreets(inputValue, addressesData);
+
+  return (
+    <SafeAreaViewFlex>
+      <ScrollView keyboardShouldPersistTaps="always">
+        <WrapperWithOrientation>
+          <Autocomplete
+            containerStyle={styles.autoCompleteContainer}
+            data={filteredStreets}
+            defaultValue={inputValue}
+            onChangeText={(text) => setInputValue(text)}
+            placeholder="Straße"
+            renderItem={renderSuggestion}
+          />
+          <View style={styles.topMarginContainer}>
+            <Calendar
+              markedDates={getMarkedDates(parsedTypesData, streetData)}
+              markingType="multi-dot"
+              renderArrow={renderArrow}
+              theme={{ todayTextColor: colors.primary }}
```

Comment:
is there a way to disable clicking on dates? i can click, there is a touchable opacity on iOS, but nothing happens. could this confuse user?

Reply:
done